packages/design-system/src/DropdownLegacy.tsx

```
@@ -0,0 +1,320 @@
+                    label={item.label}
+                    onClick={item.onClick}
+                    shorting={item.shorting}
+                    className={{
+                      root: className?.item,
+                      active: className?.activeItem,
+                    }}
+                    data={item.data}
+                    icon={item.icon}
+                    tooltip={item.tooltip}
+                    disabled={item.disabled}
+                  />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)